code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -452,32 +452,37 @@
 				to_chat(M, "<span class='notice'>That tasted horrible.</span>")
 	..()
 
-
-/datum/reagent/spraytan/overdose_process(mob/living/M)
+/datum/reagent/spraytan/overdose_start(mob/living/M)
+	. = ..()
 	metabolization_rate = 1 * REAGENTS_METABOLISM
 
-	if(ishuman(M))
-		var/mob/living/carbon/human/N = M
-		N.hair_style = "Spiky"
-		N.facial_hair_style = "Shaved"
-		N.facial_hair_color = "000"
-		N.hair_color = "000"
-		if(!(HAIR in N.dna.species.species_traits)) //No hair? No problem!
-			N.dna.species.species_traits += HAIR
-		if(N.dna.species.use_skintones)
-			N.skin_tone = "orange"
-		else if(MUTCOLORS in N.dna.species.species_traits) //Aliens with custom colors simply get turned orange
-			N.dna.features["mcolor"] = "f80"
-		N.regenerate_icons()
-		if(prob(7))
-			if(N.w_uniform)
-				M.visible_message(pick("<b>[M]</b>'s collar pops up without warning.</span>", "<b>[M]</b> flexes [M.p_their()] arms."))
-			else
-				M.visible_message("<b>[M]</b> flexes [M.p_their()] arms.")
+	if(!ishuman(M))
+		return
+	var/mob/living/carbon/human/N = M
+	N.hair_style = "Spiky"
+	N.facial_hair_style = "Shaved"
+	N.facial_hair_color = "000"
+	N.hair_color = "000"
+	if(!(HAIR in N.dna.species.species_traits)) //No hair? No problem!
+		N.dna.species.species_traits += HAIR
+	if(N.dna.species.use_skintones)
+		N.skin_tone = "orange"
+	else if(MUTCOLORS in N.dna.species.species_traits) //Aliens with custom colors simply get turned orange
+		N.dna.features["mcolor"] = "f80"
+	N.regenerate_icons()
+	..()
```

Comment:
..() gets called twice

Reply:
Looks like that first one was automatically made by VSC upon creating the overdose_start bit, will get rid of it.